be/src/runtime/stream_load/stream_load_pipe.h

```diff
@@ -110,14 +110,24 @@ class StreamLoadPipe : public MessageBodySink {
     Status _err_st = Status::OK();
 };
 
+class StreamLoadPipeReader {
+public:
+    StreamLoadPipeReader(std::shared_ptr<StreamLoadPipe> pipe) : _pipe(pipe) {}
+    virtual ~StreamLoadPipeReader() {}
+    virtual StatusOr<ByteBufferPtr> read() { return _pipe->read(); }
+
+private:
+    std::shared_ptr<StreamLoadPipe> _pipe;
+};
+
 const size_t DEFAULT_DECOMPRESS_BUFFER_SIZE = 32 * 1024 * 1024;
```

Comment:
If DEFAULT_DECOMPRESS_BUFFER_SIZE is only used in CompressedStreamLoadPipeReader, put it in.